phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -285,6 +293,11 @@ public static IndexDisableLoggingType fromValue(byte[] value) {
         , "Disable logging of failed verification rows for BEFORE, " +
         "AFTER, or BOTH verify jobs");
 
+    private static final Option USE_INDEX_TABLE_AS_SOURCE_OPTION =
+        new Option("fi", "from-index", false,
+            "To verify every row in the index table has a corresponding row in the data table. " +
+                "Only supported for global indexes. This option is applicable to BEFORE and ONLY type verification jobs");
```

Comment:
I think stating this is clearer: This option is applicable only when -v ONLY or -v BEFORE is provided.